my-profile-webapp/src/main/webapp/my-app/lec/partials/emergency-info.html

```diff
@@ -43,7 +43,7 @@ <h1>{{ alert.title }}</h1>
   <div layout="row" layout-align="center center" ng-show="profileLoadingState.ephone || profileLoadingState.lcontact || profileLoadingState.einfo">
     <loading-gif data-object="{{[]}}"></loading-gif>
   </div>
-  <div>
+  <div class="up-portlet-content-wrapper">
    <p>Students now update their local and emergency contact information in the <a href="/apps/details/student-center">Student Center</a>&nbsp;</p>
```

Comment:
These need to open in a new tab. I believe you need to add target="_blank" and rel="noopener noreferrer"